### PR TITLE
Set up a Hello Microverse project file

### PR DESCRIPTION
**Pull request by abenezer-addisu**
Setting up 'Hello Microverse' project branch
In this branch :

- Added HTML file
- Added CSS file to style the HTML
- Linters Added
- Readme file edited

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
```
Comment (abdulhamiid) [inline README.md]: - [ ] You did a good job working on setting up your first project at Microverse! 💯 However,  

- [ ] Please kindly note that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo and their contents are customized accordingly. The only optional section is FAQ.

### TEMPLATE HEADERS 👇 
image

- [ ] A README file is a guide that gives users a detailed description of a project you have worked on. so it is always necessary to have a descriptive README.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @abenezer-addisu 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,200 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```
Comment (Strangeal) [inline README.md]: Please you can remove this from the `README`. The **logo section** is supposed to be the logo of the project you're building which you clearly don't need in this project. You're going to be using it in upcoming projects on **Microverse** so

File: README.md [hunk 3]
```diff
@@ -1 +1,200 @@
...
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (Strangeal) [inline README.md]: Please remove the `Live Demo` on `line 20.` You only use it when you're building a project that you're going to deploy.

File: README.md [hunk 4]
```diff
@@ -1 +1,200 @@
...
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```
Comment (Strangeal) [inline README.md]: Please remove the `Deployment` on `line 27`. You only use it when you're building a project that you're going to deploy.

File: README.md [hunk 5]
```diff
@@ -1 +1,200 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+____
+> Describe between 1-3 key features of the application.
+
+- **[Added index.html file]**
+- **[Added style.css file]**
+- **[Added Linter]**
+
```
Comment (Strangeal) [inline README.md]: Please refactor this and add valid features of the project you're working on. Example 
Key Features
```
Use linter
Use Professional documentation
Use HTML and CSS
```

File: README.md [hunk 6]
```diff
@@ -1 +1,200 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+___
+
+- Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Strangeal) [inline README.md]: Please remove this as it's not needed in this project. You're going to need it in upcoming projects when you start working on projects that you're going to `deploy`.

File: README.md [hunk 7]
```diff
@@ -1 +1,200 @@
...
+
+## 💻 Getting Started <a name="getting-started"></a>
+_____
+
+> To get a local copy up and running, follow these steps..
+
+
+
+### Prerequisites
+
```
Comment (Strangeal) [inline README.md]: Please you can remove the `Prerequisites` section. It's not needed in this project but you will need it in upcoming projects.

File: README.md [hunk 8]
```diff
@@ -1 +1,200 @@
...
+In order to run this project you need:
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
```
Comment (Strangeal) [inline README.md]: Please include the steps a person needs to take in order for them to run your project on their local machine.
Example: 
```
  cd my-folder (replace my-folder with the name your repository)
  git clone (replace this with your github HTTP link or SSH link)
```

File: README.md [hunk 9]
```diff
@@ -1 +1,200 @@
...
+### Install
+
+ Web Browser
+
+
+
+
```
Comment (Strangeal) [inline README.md]: Please remove this as you don't need to install any `packages` to run this project.

File: README.md [hunk 10]
```diff
@@ -1 +1,200 @@
...
+### Usage
+
+To run the project, execute the following command:
+
+Code Editor
+
```
Comment (Strangeal) [inline README.md]: Please include a description of how a user can `run` the project after cloning it. Example 
**To run the project, open the `index.html` file with a browser or use a live server with your favorite code editor**

File: README.md [hunk 11]
```diff
@@ -1 +1,200 @@
...
+To run tests, run the following link:
+Run github link 
+
+
+### Deployment
+
+Comming Soon
+
+
```
Comment (Strangeal) [inline README.md]: Please remove this. It's not needed in this project

File: README.md [hunk 12]
```diff
@@ -1 +1,200 @@
...
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+___
+
+- [ ] **[Adding Nav bar]**
+- [ ] **[Adding Footer]**
+- [ ] **[Adding more Styles]**
+
```
Comment (Strangeal) [inline README.md]: [OPTIONAL] You did great 😊  adding this section. Please remove the **adding** you have before feature. You can just input only the feature. Example Navbar, Footer, UI(styles)

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @abenezer-addisu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Professional README file :heavy_check_mark: 
- All requested changes resolved :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**